### PR TITLE
[ENH] Add echo-idx flag

### PR DESCRIPTION
**Pull request by emdupre**
## Changes proposed in this pull request

Addition to #1263. Following up on discussions in #1324, adds an `echo-idx` flag to the fmriprep cli so that users can elect to preprocess only one echo in a mutliecho BOLD series.  

Currently failing due to merge conflicts in #1263.

## Documentation that should be reviewed

Usage documentation is automatically built from the CLI, so I don't think any additional documentation needs to be added ? Please let me know f there's elsewhere you'd like me to update !

### CODE REVIEW

File: fmriprep/cli/run.py [hunk 1]
```diff
@@ -77,6 +77,8 @@ def get_parser():
     #                     help='select a specific run to be processed')
     g_bids.add_argument('-t', '--task-id', action='store',
                         help='select a specific task to be processed')
+    g_bids.add_argument('--echo-idx', action='store',
```
Comment (effigies) [inline fmriprep/cli/run.py]: `type=int,`?

And maybe let's add the short option `'-e'`.

File: fmriprep/workflows/base.py [hunk 2]
```diff
@@ -56,6 +56,7 @@ def init_fmriprep_wf(subject_list, task_id, run_uuid, work_dir, output_dir, bids
         from fmriprep.workflows.base import init_fmriprep_wf
         wf = init_fmriprep_wf(subject_list=['fmripreptest'],
                               task_id='',
+                              echo_idx='',
```
Comment (effigies) [inline fmriprep/workflows/base.py]: `echo_idx=None`

**Comment by effigies** [comment]
LGTM. Minor suggestions.

@chrisfilo Is `--<ent>-idx` BIDS-Apps compliant? I see we used to have `--run-id`, but we also had `--session-id` and `--participant-label`, so I'm not sure whether there's a convention to follow here or not.

**Comment by emdupre** [comment]
Sorry about that ! Thanks for catching it, @effigies ! Updated both places.

File: fmriprep/cli/run.py (hunk 1 above)
Comment (emdupre) [reply fmriprep/cli/run.py]: The `-e` option is conflicting with the `-e` option for setting the env in `fmriprep-docker`. I'm reverting to only the long-form command, but let me know if you'd prefer a different approach !

Comment (effigies) [reply fmriprep/cli/run.py]: That seems fine. We can always reconsider later, but I agree, let's not cause a conflict.

**Pull request merged**